app/scripts/utils/show-mouse-position.js

```diff
@@ -41,6 +41,8 @@ const showMousePosition = (
   // Graphics for cursor position
   const graphics = new PIXI.Graphics();
 
+  const clearMousePosition = () => { graphics.clear(); };
```

Comment:
Why is this called `clearMousePosition`? It clears the graphics, not the mouse position.

Reply:
```suggestion
```

Leftover. Good catch.